frontend/src/pages/mock/login.page.tsx

```diff
@@ -18,6 +19,16 @@ const MockLogin: NextPage = () => {
   const [token, setToken] = useState("");
 
   useEffect(() => {
+    if (process.env.NEXT_PUBLIC_MOCK_API === "true") {
+      // When the API is mocked out, the API tokens are fake as well:
+      const mockIdsAsTokens: UsedToken[] = mockIds.map((id) => ({
+        lastUsed: 0,
+        token: id,
+        user: `${id}@example.com`,
+      }));
+      setUsedTokens(mockIdsAsTokens);
+      return;
+    }
```

Comment:
In `dev` mode, the fake login page lists API tokens previously used. When running against a mocked backend, those tokens are not valid, and we just list all the available mock IDs.